### PR TITLE
changed service list names to be more indicative of which is the default

### PR DESCRIPTION
**Pull request by paulhiggs**
for issue #82.

Service lists renamed as follows:
image

### CODE REVIEW

**Review by sofia-tsa — APPROVED** [review]

**Comment by paulhiggs** [comment]
@jpiesing could you confirm that the changes to the names of the service lists in the reference client sufficiently solve isue #82

File: backend/slepr-master.xml [hunk 1]
```diff
@@ -99,7 +99,7 @@
 			</sld:Delivery>
 		</sld:ServiceListOffering>
 		<sld:ServiceListOffering regulatorListFlag="false">
-		<sld:ServiceListName>DVB-I Reference Client DRM service list</sld:ServiceListName>
+		<sld:ServiceListName>DDRM service list</sld:ServiceListName>
```
Comment (jpiesing) [inline backend/slepr-master.xml:102]: Typo "DDRM" should be "DRM".

File: backend/slepr-master.xml [hunk 2]
```diff
@@ -79,8 +79,8 @@
 			</sld:ElectronicAddress>
 		</sld:Provider>
 		<sld:ServiceListOffering regulatorListFlag="false">
-		<sld:ServiceListName>DVB-I Reference Client service list</sld:ServiceListName>
-		<sld:ServiceListName xml:lang="zh">DVB-I 参考客户端服务列表</sld:ServiceListName>
+		<sld:ServiceListName>DVB-I Reference Client default service list</sld:ServiceListName>
```
Comment (jpiesing) [inline backend/slepr-master.xml:82]: What about adding "New users should choose this list" or something like that? "default service list" sort-of implies that but people for who English isn't their first or second language may not pick up on that.

**Review by jpiesing — COMMENTED** [review]
Consider splitting the default service list from the ones that are more specialised and putting that one at the top of the list.

**Comment by paulhiggs** [comment]
How about this - or should we drop "default" and use "Base Demonstration" since we have other lists for DRM, Prominence, Availabilty etc

image

**Comment by jpiesing** [comment]
> How about this - or should we drop "default" and use "Base Demonstration" since we have other lists for DRM, Prominence, Availabilty etc
> 
> image

Why not move "DVB default list" to the top?

**Comment by paulhiggs** [comment]
image
 If you are happy with this, I will commit the service list registry update

**Review by jpiesing — APPROVED** [review]
Looks good to me.

**Review by sofia-tsa — APPROVED** [review]

**Pull request merged**